src/main/java/it/pagopa/pn/deliverypush/sanitizers/HtmlSanitizer.java

```diff
@@ -0,0 +1,105 @@
+package it.pagopa.pn.deliverypush.sanitizers;
+
+import it.pagopa.pn.deliverypush.dto.address.LegalDigitalAddressInt;
+import it.pagopa.pn.deliverypush.dto.address.PhysicalAddressInt;
+import it.pagopa.pn.deliverypush.dto.ext.delivery.notification.NotificationPaymentInfoInt;
+import it.pagopa.pn.deliverypush.dto.ext.delivery.notification.NotificationRecipientInt;
+import org.owasp.html.PolicyFactory;
+import org.owasp.html.Sanitizers;
+import org.springframework.util.StringUtils;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Class that performs via the {@link #sanitize(Object)} method a cleanup of input parameters,
+ *  allowing only determined HTML elements, based on the policies set in the constructor via the field {@link #policy}
+ */
+public abstract class HtmlSanitizer {
+
+    private final PolicyFactory policy;
+
+    public HtmlSanitizer() {
+        this.policy = Sanitizers.IMAGES;
```

Comment:
Non sono permessi tag HTML di nessun tipo

Reply:
modificata Policy in modo tale da non permettere nessun tipo di elemento HTML